resources/plugins/simln/plugin.py

```diff
@@ -42,6 +42,7 @@ class PluginError(Exception):
 
 class PluginContent(Enum):
     ACTIVITY = "activity"
+    CAPACITY_MULTIPLIER = "capacity_multiplier"
```

Comment:
I think you meant this, because this would be the name of the field in the yaml examples?

```suggestion
    CAPACITY_MULTIPLIER = "capacityMultiplier"
```